pkg/operation/botanist/controlplane.go

```diff
@@ -292,13 +292,6 @@ func (b *Botanist) HibernateControlPlane(ctx context.Context) error {
 			return err
 		}
 
-		if err := flow.Parallel(
```

Comment:
Recently https://github.com/gardener/gardener/pull/2100 was merged to delete the dnsentries for hibernated Shoots.
After hibernation with this PR, they are not deleted:
```
$ k -n shoot--foo--bar get dnsentries
external   api.foo.bar.example.com   aws-route53   shoot--foo--bar/external   Ready    74m
internal   api.foo.bar.example.com   aws-route53   shoot--foo--bar/internal   Ready    74m
```

Could you please adapt?

Reply:
I've updated the PR with commit 2ae56c8 - now this functionality is included.